src/middleware/sentry.js

```diff
@@ -0,0 +1,108 @@
+const Sentry = require('@sentry/node');
+const { sha } = require('../helper/version');
+const { version } = require('../../package.json');
+
+/**
+ * helpers
+ */
+const replaceIds = (string) => {
+	if (string) {
+		return string.replace(/[a-f\d]{24}/ig, '[id]');
+	}
+	return string;
+};
+
+/**
+ * Middlewares
+ * @param {SentryEvent} event
+ * @param {Object(event_id, originalException, syntheticException)} hint
+ * @param {FeatherApp} app
+ * @returns {SentryEvent || undefined} Return modified sentry event, or undefined to skip sending event
+ */
+const removeIdMiddleware = (event, hint, app) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['hint' is defined but never used. Allowed unused args must match /app|req|res|next|options|params/. (no-unused-vars)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4527512)